staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -96,54 +108,21 @@ var (
 // Mandatory FlowSchema objects
 var (
 	// exempt priority-level
-	MandatoryFlowSchemaExempt = newFlowSchema(
+	MandatoryFlowSchemaExempt = NewFSAllGroups(
```

Comment:
Personally, instead of building sophisticated factory helpers, I'd vote for embedding yaml and/or json directly and unmarshalling. This has the benefit that it's copy-pastable for people who want to reuse it in other contexts.

We can always change this in a follow-up.

Reply:
I have refactored to prepare for that, no longer exporting anything besides the objects.

Reply:
pushed now